Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```
@@ -102,6 +102,7 @@ configuration:
   required: false
 - display: STIX Objects To Fetch
   name: objects_to_fetch
+  additionalinfo: 'Choose which STIX objects to fetch from the TAXII server. If left empty, all available objects will be fetched.'
```

Is it objects or types?
```suggestion
  additionalinfo: 'Choose which STIX object types to fetch from the TAXII server. If left empty, all available object types will be fetched.'
```